adaptive/tests/test_learners.py

```diff
@@ -30,6 +30,7 @@
 try:
     from adaptive.learner.skopt_learner import SKOptLearner
 except (ModuleNotFoundError, ImportError):
+    # XXX: catch the ImportError because of https://github.com/python-adaptive/adaptive/pull/278
```

Comment:
Please actually refer to an issue: a reference to a merged PR doesn't allow to track whether this todo is actionable.

Reply:
I don't think this is worth tracking in the Adaptive repo.

I could link the skopt issue directly (which is linked here too).

Reply:
Ah, sorry, overlooked that one. Let's indeed link to the skopt issue.